drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -498,6 +498,8 @@ uint8_t at86rf2xx_set_state(at86rf2xx_t *dev, uint8_t state)
         old_state = at86rf2xx_get_status(dev);
     } while (old_state == AT86RF2XX_STATE_BUSY_RX_AACK ||
              old_state == AT86RF2XX_STATE_BUSY_TX_ARET ||
+             old_state == AT86RF2XX_STATE_BUSY_RX      ||
+             old_state == AT86RF2XX_STATE_BUSY_TX      ||
```

Comment:
Shouldn't all of these be `AT86RF2XX_PHY_%`?

Reply:
`AT86rF2XX_PHY_%` maps to the corresponding TX_xxx and RX_xxx events. We could do that if we assume this function is hardcoded to either basic or extended mode (that is, a user of the basic mode would not try to access an extended state by any means)

Reply:
Ok I undestand I think I confused the asignment when reading the first time.